### PR TITLE
Fix round1

### PR DESCRIPTION
**Pull request by pb-cdunn**
I've rebased this. It has the essential fix, plus comments etc.

See also #57, originally on this branch.

### CODE REVIEW

**Comment by pb-cdunn** [comment]
Old:
```
final idx contains: 298 forward and reverse SV kmers.
before:298 maxRefCount:1
after:44 maxRefCount:1
```
New:
```
final idx contains: 270 forward and reverse SV kmers.
before:270 maxRefCount:1
after:54 maxRefCount:1
```
Diffs in `output.vcf`:
```diff
229c229
< chr22	48910763	pbsv.DEL.66387	CCCCAGATTCTGAAATCTTTCATTGTGGTTGAAGTCTCCCCTCCCGA	C	.PASS	SVTYPE=DEL;END=48910809;SVLEN=-46;SVANN=TANDEM;NIB_SV_REF_KMERIDX_COUNT=0;NIB_SV_ALT_KMERIDX_COUNT=0	GT:AD:DP	0/1:17,11:28	0/1:16,13:29	0/0:25,0:25
---
> chr22	48910763	pbsv.DEL.66387	CCCCAGATTCTGAAATCTTTCATTGTGGTTGAAGTCTCCCCTCCCGA	C	.PASS	SVTYPE=DEL;END=48910809;SVLEN=-46;SVANN=TANDEM;NIB_SV_REF_KMERIDX_COUNT=54;NIB_SV_ALT_KMERIDX_COUNT=54;NIB_SAMPLES_WITH_SV=test;NIB_READ_SUPPORTS=26	GT:AD:DP	0/1:17,11:28	0/1:16,13:29	0/0:25,0:25
```

File: src/nibpkg/reporter.nim [hunk 1]
```diff
@@ -38,10 +43,16 @@ proc report*(vcf_name : string, sv_read_supports : CountTableRef[uint32], sv_ind
         var sv_support_count = sv_read_supports.getOrDefault(sv_id, -1)
         var sv_ref_k_count = 0
         var sv_alt_k_count = 0
-        if sv_id in sv_index.counts:
-            for km in sv_to_kmer.getOrDefault(sv_id):
-              sv_ref_k_count += sv_index.counts[km].refCount.int
-              sv_alt_k_count += sv_index.counts[km].altCount.int
+        # if sv_id in sv_index.counts:
```
Comment (zeeev) [inline src/nibpkg/reporter.nim]: let's cleanup these left over comments, and commented out echos?

File: src/nibpkg/reporter.nim [hunk 2]
```diff
@@ -14,11 +14,16 @@ proc report*(vcf_name : string, sv_read_supports : CountTableRef[uint32], sv_ind
...
-        var a = sv_to_kmer.getOrDefault(sv)
+      doAssert(support.svs.len != 0)
+      for svId in support.svs:
+        var a = sv_to_kmer.getOrDefault(svId)
         a.add(kmer)
-        sv_to_kmer[sv] = a
+        sv_to_kmer[svId] = a
+        #echo a
+    echo "sv to kmer:"
+    echo sv_to_kmer
```
Comment (zeeev) [inline src/nibpkg/reporter.nim]: This will be a doozy, let's remove this echo.

**Comment by zeeev** [comment]
Thanks @pb-cdunn

**Pull request merged**